lib/messages.js

```diff
@@ -7,6 +7,6 @@ exports.emptyCache = function() {
 
 exports.notFound = function() {
   return 'Page not found\n' +
-  'Try updating with --update, or submit a pull request to \n' +
+  'Try updating with tldr --update, or submit a pull request to:\n' +
```

Comment:
Good idea, maybe even `"tldr --update"` (with quotes) to show that it's a command?